docs/installation.md

```diff
@@ -0,0 +1,41 @@
+# Installation
+
+## Editor-handled Installation
+
+Some editors have built-in logic to install and update the language server automatically, so you typically shouldn't need to worry about installation or updating of the server in these editors, as long as you use the linked extension.
```

Comment:
```suggestion
Most text editor extensions/plugins have built-in logic to install and update language servers automatically, so you typically shouldn't need to worry about installation or updating of the server, as long as you use the provided extension.
```

Reply:
I thought about saying "most" but it would only be true if you said "most of the most commonly used with this LS" and that's also only relying the (lack of) data we have and I just didn't want to go down the road of implying anything about the user base size in this particular context.